README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
 
----
-# How can I start?
+If the ARG later finds that such an AI needs significant work, the ARG might vote that the AI requires the creation of a Language Study Group to proceed. At that point the original owner of the original issue is encouraged to lead that effort.
 
-The best way to get started is to browse open issues to get a sense of how issues are structured, and to see if you find any existing issues that interest you. Consider speaking with supporters within that discussion about proposing a Language Study Group to take the issue further.
+## Language Study Groups
 
-If there is an issue that you have not found discussed, please be welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
+Language Study Groups are created to transparently capture the work towards the generation and ongoing development of an AI to eventually be deliberated and voted on by the ARG.
 
-# Ada Interpretations (AIs)
+Language Study Groups can be created through two mechanisms. The first is by ARG vote to require a Language Study Group be created to further develop an existing AI. The second is by a direct proposal to form a Language Study Group, submitted to the ARG by an individual or group of the larger Ada community. 
 
-The ARG performs most of its work in terms of Ada Interpretations (AIs), which are documents that describe an identified issue with the Ada Reference Manual, or which propose enhancements to the Ada programming language.  Such issues may be identified within the ARG itself, or may be identified by the wider Ada community.  Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements.  
+Language Study Groups proposed by the community must be approved by the ARG. Once approved, those proposing the creation of the study group can work towards the creation of a new AI for formal consideration by the ARG. Language Study Group proposals from the community should be for significant changes or new features that will require rigorous design and prototyping work. 
 
-Language Study Groups work towards the development of an AI that can be submitted to the ARG for formal consideration.
+Requests to form a new Language Study Group can be submitted through the official ARG [web-form](https://arg.adaic.org/community-input).
 
-AIs that are being being active considered by the ARG will be posted in this repository in the AI format. This is great reference for language study groups to understand how AIs should be formed, and what they should contain.
+It is generally encouraged that Language Study Groups be formed of group of interested individuals, but that is not a formal requirement.
```

Comment:
I am confused by the phrase "encouraged that LSGs be formed of group of interested individuals".  What other sort of group would form an LSG?  I don't understand the point of this sentence.  Are you implying it might be a single individual?  Also, the sentence is a bit awkward.  If we think we need the sentence (I am not convinced), then perhaps:
"... encouraged that LSGs be formed by two or more interested individuals, but a single individual can create an LSG if that makes sense."

Reply:
I suspect that the point as lost here (but I'm not certain what it was). It seems to me that any functional LSG is going to be created and run by a particular engaged individual that will drive the various steps to getting a result. In the absence of that, it will be like ARG homework, lots of good intentions, not much results. (Before I took over, little was happening, especially in the upkeep of AIs. I still have to crack the whip periodically.)

So a group needs a committed leader. It makes sense to encourage a number of other people as well. Not sure if it makes sense to say more.

Reply:
Wording improved.